vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -26,22 +26,23 @@ package vmdk
 
 import (
 	"fmt"
-	"path/filepath"
-	"sync"
-	"time"
-
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/go-plugins-helpers/volume"
 	"github.com/vmware/docker-volume-vsphere/vmdk_plugin/drivers/vmdk/vmdkops"
 	"github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/fs"
 	"github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/refcount"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
 )
 
 const (
 	devWaitTimeout   = 1 * time.Second
 	sleepBeforeMount = 1 * time.Second
 	watchPath        = "/dev/disk/by-path"
 	version          = "vSphere Volume Driver v0.4"
+	recvError        = "Receive error"
```

Comment:
This constant looks out of place here. Do we have separate declaration of error codes anywhwre?